tests/framework/console/widgets/TableTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yiiunit\framework\console;
+
+use yii\console\widgets\Table;
+use yiiunit\TestCase;
+
+/**
+ * @group console
+ */
+class TableTest extends TestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->mockApplication();
+    }
+
+    public function testTable()
+    {
+        $table = new Table();
+
+        $expected = <<<EXPECTED
+╔═══════════════╤═══════════════╤═══════════════╗
+║ test1         │ test2         │ test3         ║
+╟───────────────┼───────────────┼───────────────╢
+║ testcontent1  │ testcontent2  │ testcontent3  ║
+╟───────────────┼───────────────┼───────────────╢
+║ testcontent21 │ testcontent22 │ testcontent23 ║
+╚═══════════════╧═══════════════╧═══════════════╝
+
+EXPECTED;
+
+        $this->assertEqualsWithoutLE($expected, $table->setHeaders(['test1', 'test2', 'test3'])
+            ->setRows([
+                ['testcontent1', 'testcontent2', 'testcontent3'],
+                ['testcontent21', 'testcontent22', 'testcontent23']
+            ])->render()
+        );
+
+        // test fulwidth chars
+        // @see https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms
+        $expected = <<<EXPECTED
```

Comment:
I'd create a separate method such as `testTableWithFullwidthChars()` for this case.